dietpi/dietpi-backup

```diff
@@ -478,7 +478,7 @@ _EOF_
 					FP_TARGET=$(</tmp/dietpi-drive_manager_selmnt)
 					rm /tmp/dietpi-drive_manager_selmnt
 
-					[[ $FP_TARGET == '/' ]] && FP_TARGET='/mnt'
+					$FP_TARGET || [[ $FP_TARGET == '/' ]] && FP_TARGET='/mnt'
```

Comment:
This would execute what was ever entered as command. You want to replace it with `/mnt` if the input was empty as well? That would be:
```
[[ ! $FP_TARGET || $FP_TARGET == '/' ]] && FP_TARGET='/mnt'
```

But actually, I think it's reasonable to not apply any choice when user cancels it, instead return to the menu. Let me see ich `/boot/dietpi/dietpi-drive_manager 1` returns an ext code on cancel, so that we can check that directly.